tools/cli/utils.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable */
```

Comment:
Do you anticipate adding more to this file in the future? I think the filename could be more indicative of what this file does.

Reply:
I made this at some point because I had written more than one script and didn't like duplicate code, but since it's just the one prerelease.js script now, I think I can just move these into that file.

Reply:
I moved the functions to the prerelease.js script